mkdocs.yml

```diff
@@ -5,11 +5,24 @@ repo_name: alandefreitas/small
 
 site_description: "Small: C++ Small Containers"
 
+nav:
+  - Home: index.md
+  #  - Quickstart: quickstart.md todo
+  #  - Motivation: motivation.md todo
+  - Reference: # Reference is organized by module
+      - Small: reference/Modules/index.md        # futures module
+      - Index: reference/Namespaces/namespacesmall.md      # The futures namespace contains everything
+      - Classes: reference/Classes/index.md                  # All Classes
+      - Files: reference/Files/index.md                      # All Files
+  # - Contributing: contributing.md todo
+  # - Acknowledgments: acknowledgments.md todo
+
+
 theme:
   name: material
   custom_dir: docs/overrides
```

Comment:
The overrides are missing (or were removed?). This gives us an error:

```
ERROR    -  Config value: 'theme'. Error: The path set in theme.custom_dir ('/home/small/docs/overrides') does not exist.
```

We can get these overrides from the futures project.

Reply:
Yes, for sure. I've put it before, but I'm not quite sure about what I did to get rid of this file, lol. I suppose that when I was trying to fix the branches' organization, I made some mistakes, and that was it.